packages/node/src/utils/ensureIsWrapped.ts

```diff
@@ -24,5 +24,10 @@ export function ensureIsWrapped(
         );
       }
     });
+
+    getCurrentScope().setContext('Instrumentation', {
```

Comment:
This should be `getGlobalScope()`, I'd say - the current scope is very "volatile", putting this on the global scope ensures it will always be there!

Reply:
Was not sure about the scope, for the case if we have multiple frameworks in a project, would they overwrite each other?

Reply:
yes, the last would win! IMHO that's probably OK, but maybe you can also check with other folks on the team what they think about this :)